windows/src/desktop/kmshell/install/UfrmInstallKeyboardLanguage.pas

```diff
@@ -294,41 +261,81 @@ procedure TfrmInstallKeyboardLanguage.editSearchChange(Sender: TObject);
   FillLanguageGrid;
 end;
 
+procedure TfrmInstallKeyboardLanguage.editSearchKeyDown(Sender: TObject;
```

Comment:
Small polish: lets the user move up and down the list with arrow keys, while still in the search box.